content/concepts/contribute/community.md

```diff
@@ -9,6 +9,7 @@ aliases:
 
 We love talking about libp2p, and we'd be happy to have you in the mix.
 
-Visit our [discussion forums](https://discuss.libp2p.io) to ask questions about [using libp2p](https://discuss.libp2p.io/c/users), discuss exciting [research](https://discuss.libp2p.io/c/research), and [stay up to date with important announcments](https://discuss.libp2p.io/c/news).
+Visit our [discussion forums](https://discuss.libp2p.io) to ask questions about [using libp2p](https://discuss.libp2p.io/c/users), discuss exciting [research](https://discuss.libp2p.io/c/research), and [stay up to date with important announcements](https://discuss.libp2p.io/c/news).
 
-We also often hang out on [Discord](https://discord.com/invite/ipfs) and [IRC](irc://irc.freenode.org/%23libp2p). While Discord and IRC are great places for live chats, remember that discussions can get lost in a sea of messages. For long-lived discussion & support, [the discussion forums](https://discuss.libp2p.io) are the way to go.
+We also often hang out on [Discord](https://discord.com/invite/ipfs) and in the #libp2p-implementers channel in the [Filecoin slack](filecoinproject.slack.com
```

Comment:
Should we remove this as well? We don't hang out on that subpar messenger any more.

Reply:
Discord? Let's keep it, it's bridged to our slack channel so we are able to respond to messages from there.